CairoFCCOS/FCCOSEth004.xml

```diff
@@ -0,0 +1,246 @@
+                     </msItem>
+
+                     <msItem xml:id="ms_i5">
+                        <locus from="114" to="141"/>
+                        <title type="complete" ref="LIT1113Anqasa"/>
+                         <notatedMusic><desc>Contains musical notation.</desc></notatedMusic>
+                         <textLang mainLang="gez"/>
+                     </msItem>
+
+                        <msItem xml:id="ms_i6">
+                            <locus from="141" to="175"/>
+                            <title type="complete" ref="LIT3118RepCh398"/>
+                            <textLang mainLang="gez"/>
+                            <incipit xml:lang="gez">ፈቀደ፡ እግዚእ፡ ለአዳም፡ ያግዕዞ፡ ሥጋኪ፡ ንጹሐ፡</incipit>
+                        </msItem>
+                        
+                        <msItem xml:id="ms_i7">
+                            <locus from="175" to="178"/>
+                            <title type="incomplete" ref="LIT2960RepCh241"/>
+                            <note>Incomplete at the end.</note>
+                            <textLang mainLang="gez"/>
+                            <incipit xml:lang="gez">ንጽሕት፡ ወብርህት፡ በልሳነ፡ ኵሉ፡ <gap reason="omitted"/> ወነፍስ፡</incipit>
+                        </msItem>
+                        
+                        <msItem xml:id="ms_i8">
+                            <locus from="179" to="216"/>
+                            <title type="complete" ref="LIT6489SchoolChants#Salast">Śalast chants</title>
+                            <notatedMusic><desc>Contains musical notation.</desc></notatedMusic>
+                            <textLang mainLang="gez"/>
+                        </msItem>
+                        
+                        <msItem xml:id="ms_i9">
+                            <locus from="217" to="219"/>
+                            <locus from="220" to="266"/>
+                            <title type="incomplete" ref="LIT3100RepCh380" cert="low"/> 
+                            <note>Incomplete at the end. The text is only described as ‘Malkǝʾa Giyorgis’ in the catalogue, the identification 
+                                with <ref type="work" corresp="LIT3100RepCh380"/> is tentative.  <locus from="220" to="266"/> are located 
+                            before <locus target="#179"/>.</note>
+                            <textLang mainLang="gez"/>
+                        </msItem>
+                    </msContents>
+                    
+                    <physDesc>
+                        <objectDesc form="Codex">
+                            <supportDesc>
+                                <support>
+                                    <material key="parchment"/>
+                                    <material key="paper"/>
+                                </support>
+                                <extent>
+                                    <measure unit="page" quantity="266">266</measure>
+                                    <dimensions type="outer" unit="cm">
```

Comment:
Yes, as I wrote in a previous PR, I felt uncomfortable to convert because that would pretend to a preciseness that might not be in the original measure. Maybe a better compromise would be to convert to mm, but indicate in a note that this is not the original measure.

Reply:
I would not bother about this. We only convert the data provided in the catalogues (the source is indicated), and the conversion is precise. Otherwise, measuring a manuscript is a small issue in itself, esp. the Ethiopian ones, as measuring is always not precise. Several persons measuring the same book would provide slightly different numbers :) So, for me a slight imprecision/inconsistency is an inherent quality of the manuscript. The description should leave space for that.